extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -571,18 +601,21 @@ export class ConsentUI {
    */
   resetIframe_() {
     const {classList} = this.parent_;
-
-    // Remove the iframe active to go back to our normal height
     classList.remove(consentUiClasses.iframeActive);
+    if (this.modalEnabled_) {
+      classList.remove(consentUiClasses.modal);
+      classList.remove(consentUiClasses.borderEnabledModal);
+    } else if (this.borderEnabled_) {
+      classList.remove(consentUiClasses.borderEnabled);
+    }
```

Comment:
Nits:

1. Removing classes is fine when they're not present. You don't need to check these flags.

2. As mentioned above, the compound classname for `borderEnabledModal` is redundant since you can compound the selector, so simply:

```suggestion
    classList.remove(consentUiClasses.modal);
    classList.remove(consentUiClasses.borderEnabled);
```